pkg/types/rekord/v0.0.1/entry.go

```diff
@@ -100,6 +102,19 @@ func (v V001Entry) IndexKeys() []string {
 		}
 	}
 
+	// if this is a certificate, add email addresses to result
```

Comment:
would it be better to attempt to typecast `v.keyObj` to pki/x509/PublicKey and just use the value we've already parsed? The struct fields aren't exported at the moment but I don't see why they couldn't be. 

I'd rather handle this at the time the keyObj is created rather than re-parse at this point.

Reply:
done